release_notes/40-release-notes.adoc

```diff
@@ -0,0 +1,447 @@
+- The PDF export feature in the Vulnerability Management section of the {product-title-short} web portal is deprecated and is planned for removal in the {product-title-short} 4.2.0 release. Use the vulnerability reporting feature instead for more comprehensive CSV data.
+- Vulnerability scanning support for non-RPM (ASP).NET Core Runtime is deprecated and is planned for removal in the {product-title-short} 4.2.0 release.
+- All `/v1/report` APIs for creating and managing vulnerability reports are deprecated and will be replaced with new `/v2/report` APIs in a future release.
+
+[id="remove-kernel-rn40"]
+==== Remove kernel module as collection method
+
+Currently, secured clusters can specify three options of collection methods for runtime events: eBPF (selected by default), kernel module, or no collection. Kernel module as a collection method was deprecated in the {product-title-short} version 3.74 release and is planned for removal in the {product-title-short} version 4.1 release.
+
+[discrete]
+[id="actions-kernel-module-rn40"]
+===== Actions you must take
+Verify the collection method of your secured clusters. This value is set in the `collector.collectionMethod` parameter and is one of the following methods:
+
+- `EBPF`
+- `KERNEL_MODULE`
+- `NO_COLLECTION`
+
+If any of your secured clusters uses `KERNEL_MODULE` as a collection method, change it to `EBPF`.
+
+[id="removed-features-rn40"]
+== Removed features
+
+The following section provides additional information about removed features listed in the preceding table.
+
+[id="permissions-rn40"]
+=== Permissions and permission sets
+
+As announced in the Release Notes for {product-title-short} 3.73.0, some permissions for permission sets are now grouped for simplification. The deprecation process will remove and replace the deprecated permissions with the replacing permission in the following list. The access level granted to the replacing permission will be the lowest among all access levels of the replaced permissions.
+
+- Permission `Administration` replaces the deprecated permissions `AllComments`, `Config`, `DebugLogs`, `NetworkGraphConfig`, `ProbeUpload`, `ScannerBundle`, `ScannerDefinitions`, `SensorUpgradeConfig`, and `ServiceIdentity`.
+- Permission Compliance replaces the deprecated permission `ComplianceRuns`.
+- The `Analyst` permission set will change behavior. Instead of allowing `read` access to all resources except `DebugLogs`, it will allow `read` access to all resources except `Administration`. If you were using the `Analyst` role or permission set for actions requiring `read` access on `AllComments`, `Config`, `NetworkGraphConfig`, `ProbeUpload`, `ScannerBundle`, `ScannerDefinitions`, `SensorUpgradeConfig` or `ServiceIdentity` resources, then preemptively create a new permission set with `read` access on the `Administration` and other required resources, and reference it instead of `Analyst` in the created roles.
```

Comment:
Last sentence is kind of long, and maybe the mid-sentence list of resources could benefit from rewriting / breaking up the sentence with an unordered list.